packages/firebase_vertexai/firebase_vertexai/lib/src/api.dart

```diff
@@ -108,6 +108,15 @@ final class GenerateContentResponse {
   Iterable<FunctionCall> get functionCalls =>
       candidates.firstOrNull?.content.parts.whereType<FunctionCall>() ??
       const [];
+
+  /// The inline data parts of the first candidate in [candidates], if any.
+  ///
+  /// Returns an empty list if there are no candidates, or if the first
+  /// candidate has no [InlineDataPart] parts. There is no error thrown if the
+  /// prompt or response were blocked.
+  Iterable<InlineDataPart> get inlineDatas =>
```

Comment:
Could this be called `inlineDataParts`? That's the name we've used in the other SDKs.

Reply:
Updated